learn/advanced/filtering.mdx

```diff
@@ -164,6 +166,48 @@ NOT release_date EXISTS
 
 Both forms are equivalent.
 
+#### `IS EMPTY`
+
+The `IS EMPTY` operator selects documents in which the specified attribute exists but contains empty values. The following expression only returns documents with an empty `overview` field:
+
+```
+overview IS EMPTY
+```
+
+`IS EMPTY` matches the following JSON values:
+
+- `""`
+- `[]`
+- `{}`
+
+Meilisearch does not treat `null` values as empty. To match `null` fields, use the [`IS NULL`](#is-null) operator.
+
+Use `NOT` to build the negated form of `IS EMPTY`:
+
+```
+attribute IS NOT EMPTY
+NOT attribute IS EMPTY
```

Comment:
For consistency, can we use a[ real attribute](https://www.meilisearch.com/docs/learn/advanced/filtering#configuring-filters:~:text=the%20following%20fields%3A-,JSON,-%5B%0A%20%20%7B%0A%20%20%20%20%22id) instead of "attribute"?

Reply:
Duh, I initially used "overview", but then changed it when I was reviewing because I was comparing it with the spec.